.github/workflows/job-precommit-python.yml

```diff
@@ -0,0 +1,275 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PreCommit Python
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+  pull_request_target:
+    branches: [ 'master', 'release-*']
+    tags: 'v*'
+    paths:
+      - 'sdks/python/apache_beam/**'
+      - 'sdks/python/apache_beam/runners/worker/**'
+      - 'sdks/python/apache_beam/testing/**'
+      - 'sdks/python/apache_beam/transforms/**'
+      - 'sdks/python/apache_beam/typehints/**'
+      - 'sdks/python/apache_beam/utils/**'
+      - 'sdks/python/apache_beam/coders/**'
+      - 'sdks/python/apache_beam/internal/**'
+      - 'sdks/python/apache_beam/metrics/**'
+      - 'sdks/python/apache_beam/options/**'
+      - 'sdks/python/apache_beam/runners/portability/fn_api_runner/**'
+      - 'sdks/python/apache_beam/runners/portability/stager_test.py'
+      - 'sdks/python/apache_beam/runners/runner_test.py'
+      - 'sdks/python/apache_beam/runners/sdf_utils_test.py'
+      - 'sdks/python/apache_beam/tools/microbenchmarks_test.py'
+      - 'sdks/python/apache_beam/io/concat_source_test.py'
+      - 'sdks/python/apache_beam/io/external/generate_sequence_test.py'
+      - 'sdks/python/apache_beam/io/iobase_test.py'
+      - 'sdks/python/apache_beam/io/range_trackers_test.py'
+      - 'sdks/python/apache_beam/io/restriction_trackers_test.py'
+      - 'sdks/python/apache_beam/io/source_test_utils_test.py'
+      - 'sdks/python/apache_beam/io/sources_test.py'
+      - 'sdks/python/apache_beam/io/utils_test.py'
+      - 'sdks/python/apache_beam/io/watermark_estimators_test.py'
+      - 'sdks/python/apache_beam/runners/common_test.py'
+      - 'sdks/python/apache_beam/runners/pipeline_context_test.py'
+      - 'sdks/python/apache_beam/runners/portability/artifact_service_test.py'
+      - 'sdks/python/apache_beam/runners/portability/job_server_test.py'
+      - 'sdks/python/apache_beam/runners/portability/local_job_service_test.py'
+      - 'sdks/python/apache_beam/runners/portability/portable_runner_test.py'
+      - 'sdks/python/apache_beam/runners/portability/sdk_container_builder_test.py'
+      - 'sdks/python/apache_beam/io/hadoopfilesystem_test.py'
```

Comment:
Could you please sort these paths alphabetically? It will make it much easier to read/edit

Reply:
Done 👍